staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -77,6 +77,12 @@ type KubeSchedulerConfiguration struct {
 	// the default value (10s) will be used.
 	PodMaxBackoffSeconds *int64 `json:"podMaxBackoffSeconds,omitempty"`
 
+	// PodMaxInUnschedulablePodsDuration is the maximum time a pod can stay in
+	// unschedulablePods. If a pod stays in unschedulablePods for longer than this
+	// value, the pod will be moved from unschedulablePods to backoffQ or activeQ.
```

Comment:
Might get off the subject, is it possible to create a testcase to simulate the scenario that pod is moved from the unschedulablePods to backoffQ or activeQ if the pod stays in unschedulablePods for longer than `PodMaxInUnschedulablePodsDuration`?

Reply:
We already have test cases about this in https://github.com/kubernetes/kubernetes/blob/0852a2759afad8dce8f63ed7b5be2859084996d1/pkg/scheduler/internal/queue/scheduling_queue_test.go#L1416-L1417

Reply:
Ack.